src/orderbook.js

```diff
@@ -1062,28 +1188,33 @@ OrderBook.prototype.deleteOffer = function(node, isOfferCancel) {
 OrderBook.prototype.setOffers = function(offers) {
   assert(Array.isArray(offers), 'Offers is not an array');
 
-  const self = this;
-
   this.resetCache();
 
-  const newOffers = _.map(offers, function(rawOffer) {
-    const offer = OrderBook.offerRewrite(rawOffer);
+  let i = -1, offer;
+  const l = offers.length;
```

Comment:
I think the common rule is that you can only use one-letter variable names for loop indices or mathematical symbols.